src/SosThreadingTools/SosThreadingTools.csproj

```diff
@@ -1,15 +1,18 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>
-    <TargetFrameworks>net472</TargetFrameworks>
+    <TargetFrameworks>net7.0-windows</TargetFrameworks>
     <RuntimeIdentifiers>win-x86;win-x64</RuntimeIdentifiers>
     <ExtrasBuildEachRuntimeIdentifier>true</ExtrasBuildEachRuntimeIdentifier>
+    <EnableDynamicLoading>true</EnableDynamicLoading>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
 
-    <AssemblyName>$(MSBuildProjectName)</AssemblyName>
-    <DebugSymbols>true</DebugSymbols>
-    <DllExportOurILAsm>false</DllExportOurILAsm>
+    <!-- This project's build is incomplete without DNNE, which only works on Windows. -->
```

Comment:
This isn't true. DNNE works on all platforms but uses the TFM to understand what platform to target. This is a fundamental difference between unmanaged and managed code and nothing to do with DNNE. Setting the TFM to net7.0 would permit this to build.

Reply:
I've modified the comment to specify that DNNE only works on Windows because of our -windows TFM.